native/Avalonia.Native/src/OSX/app.mm

```diff
@@ -1,8 +1,39 @@
 #include "common.h"
+#include <Cocoa/Cocoa.h>
+#import <objc/runtime.h>
+
+@implementation NSBundle (FakeBundleIdentifier)
+
+- (NSString *)__bundleIdentifier;
+{
+    if (self == [NSBundle mainBundle]) {
+        return @"com.avalonia.native.osx";
```

Comment:
We should add an option to configure that with `AvaloniaNativeMacOptions` and fall back to ``com.avaloniaui.apps.`+Assembly.GetEntryAssembly().Name`

Reply:
This probably should also somehow interact with bundle identifier from Info.plist if the app is running from an actual app bundle.

Reply:
I guess we should check the `[[NSBundle mainBundle] bundleIdentifier]` and only install the fake bundle identifier when there isn't.